electron/main.js

```diff
@@ -26,24 +39,52 @@ function createWindow() {
   );
 
   // Open the DevTools.
-  win.webContents.openDevTools();
-
+  if (isDevMode) {
+    win.webContents.openDevTools();
+  }
   // Emitted when the window is closed.
   win.on('closed', () => {
     // Dereference the window object, usually you would store windows
     // in an array if your app supports multi windows, this is the time
     // when you should delete the corresponding element.
     win = null;
   });
+
+  win.webContents.on('did-finish-load', () => {
+    // Windows: Handle deeplink url
+    if (process.platform == 'win32') {
+      // Keep only command line / deep linked arguments
+      deeplinkingUrl = process.argv.slice(1);
+    }
+    if (deeplinkingUrl) {
+      setTimeout(() => {
+        win.webContents.send('open-url-event', deeplinkingUrl);
+      }, 1000);
```

Comment:
shouldn't the event 'did-finish-load' be the right timing to send 'open-url-event' ? Why the extra 1000 ms?

Reply:
The problem here is that is not yet subscribed to the 'open-url-event' event, this takes place after the profile loaded. And we need the profile loaded and all the wallets to handle de URL correctly.